src/aliceVision/image/io.cpp

```diff
@@ -23,12 +24,76 @@
 #include <iostream>
 #include <cmath>
 
-
 namespace fs = boost::filesystem;
 
 namespace aliceVision {
 namespace image {
 
+std::string EImageColorSpace_informations()
+{
+    return EImageColorSpace_enumToString(EImageColorSpace::AUTO) + ", " +
+           EImageColorSpace_enumToString(EImageColorSpace::LINEAR) + ", " +
+           EImageColorSpace_enumToString(EImageColorSpace::SRGB) + ", " +
+           EImageColorSpace_enumToString(EImageColorSpace::ACES) + ", " +
+           EImageColorSpace_enumToString(EImageColorSpace::ACEScg) + ", " +
+           EImageColorSpace_enumToString(EImageColorSpace::NO_CONVERSION);
+}
+
+EImageColorSpace EImageColorSpace_stringToEnum(const std::string& dataType)
+{
+    std::string type = dataType;
+    std::transform(type.begin(), type.end(), type.begin(), ::tolower); // tolower
```

Comment:
```suggestion
    std::string type = boost::to_lower_copy(dataType);
```

Reply:
`const std::string type` in that case

Reply:
If `const` added, line 108 also needs to be updated to keep coherency.